cypress/support/step-definitions/common-steps.js

```diff
@@ -532,6 +532,24 @@ When('I press {word} on focused element', (key) => {
   cy.focused().trigger('keydown', { key });
 });
 
+When('I press {string} onto focused element', (arrow) => {
+  switch (arrow) {
+    case 'downarrow':
+      cy.focused().trigger('keydown', { keyCode: 40, which: 40 });
+      break;
+    case 'uparrow':
+      cy.focused().trigger('keydown', { keyCode: 38, which: 38 });
+      break;
+    case 'enter':
+      cy.focused().trigger('keydown', { keyCode: 13, which: 13 });
+      break;
+    case 'space':
+      cy.focused().trigger('keydown', { keyCode: 32, which: 32 });
+      break;
+    default: throw new Error('Could not press that arrow key');
```

Comment:
do you know other than mentioned above arrows? :)
```suggestion
    default: throw new Error(`This key ${arrow} is not one of the arrow ones`);
```

Reply:
fixed